lib/soccerama/client/commentaries.rb

```diff
@@ -2,7 +2,7 @@ module Soccerama
   class Client
     module Commentaries
       def commentaries(match_id)
-        get("/commentaries/match/#{match_id}")['commentaries']
+        get("/commentaries/match/#{match_id}").fetch('commentaries')
```

Comment:
No worries! Feel free to communicate with me on facebook or email at junster90@gmail.com